test/core/modules/invert.js

```diff
@@ -11,7 +11,7 @@ var base64Img = require('base64-img');
 require('../../../src/ImageSequencer.js');
 
 //require image files as DataURLs so they can be tested alike on browser and Node.
-var sequencer = ImageSequencer({ ui: false });
+var sequencer = ImageSequencer({ ui: false});
```

Comment:
This is such a simple change maybe we should revert it because it creates a diff line?

Reply:
Ok sure!!